_sass/_mixins.scss

```diff
@@ -19,8 +19,13 @@
 .navbar-btn {
   color: $white !important;
   background-color: none !important;
+  &:focus {
+    color: $white !important;
+    background-color: $primary !important;
```

Comment:
In your branch, this:
![image](https://user-images.githubusercontent.com/4673363/136874556-4abdac69-d872-4758-b082-6d951e6b3207.png)
...now appears like this:
![image](https://user-images.githubusercontent.com/4673363/136874642-22805c1c-05b9-414d-a280-c6f3f31f93dc.png)

Was that intentional?

Reply:
Currently the focus is a very faint border like this:

![image](https://user-images.githubusercontent.com/4673363/137052277-3dc2610a-7e4f-450b-8de9-d0d00629de84.png)

...and it does render around the highlighted `Playground` button (although very difficult to discern):

![image](https://user-images.githubusercontent.com/4673363/137052341-2744420d-5b86-4a84-9ba5-df5ab91dcfdb.png)

I noticed that the https://www.microsoft.com handles this by using underlines instead of borders.  So maybe we could use the same solution, something like this:

![image](https://user-images.githubusercontent.com/4673363/137053368-d83d140a-34bc-4920-8392-263405bd3a29.png)

It is a slightly bigger change to the template, but very easy to apply to the other Rush Stack websites.  What do you think?